robotbase/src/main/java/com/systemmeltdown/robot/subsystems/StorageSubsystem.java

```diff
@@ -122,16 +122,16 @@ public double getMotorCurrent() {
         return m_rotateMotor.getStatorCurrent();
     }
 
-    public boolean isAlignedForShooting() {
-        final double encoderOffsetDegrees = m_throughBoreEncoder.get();
-        if (encoderOffsetDegrees > 0) {
-            return encoderOffsetDegrees < m_config.m_encoderAlignZoneDegrees;
-        } else if (encoderOffsetDegrees < 0) {
-            return encoderOffsetDegrees > -m_config.m_encoderAlignZoneDegrees;
-        } else {
-            return true;
-        }
-    }
+    // public boolean isAlignedForShooting() {
```

Comment:
It's generally not good form to merge blocks of commented out code.  If this is necessary for the time being can you at least put a comment as to why so that we know what needs to be done with this?